packages/sources/alphachain/src/adapter.ts

```diff
@@ -1,33 +1,11 @@
-import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Builder } from '@chainlink/ea-bootstrap'
 import { Config, ExecuteWithConfig, ExecuteFactory } from '@chainlink/types'
-import { makeConfig, DEFAULT_ENDPOINT } from './config'
+import { makeConfig } from './config'
 import { dataQuery } from './endpoint'
```

Comment:
It would be cleaner to do something like:
```
import * as endpoints from './endpoint'
```
and then on like 8 pass `endpoints` instead of `chainlinkEndpoints`.

This should work most places except in adapters where there were types thrown in `/endpoint/index.ts`.
For those we should move the types out of there to their own file called `types.ts`.
Eventually we will manage our types better, but this will be a quick path forward.

Reply:
So I made this change, and it breaks `buildSelector` because it changes the `endpoint` object from something like this: ```      { supportedEndpoints: [ 'price' ], execute: [AsyncFunction: execute] }``` to ```{
        price: {
          supportedEndpoints: [ 'price' ],
          execute: [AsyncFunction: execute]
        }
      }```. The difference in `buildSelector` would be that we need to index into that `price` key in the object. I'm not sure whether I should just try to query the object for the endpoints so that I can index into it, or do something where we have specific endpoint types that are iterated through.  Either way, I'm not sure if either are more efficient than the original way so we might just want to keep it how it is? Let me know what you think.